app/js/components/LabResultEntry.jsx

```diff
@@ -61,6 +61,7 @@ export class LabResultEntry extends PureComponent {
       dispatch(patientAction.getPatient(state.patient.uuid));
       dispatch(constantsActions.fetchLabResultsDidNotPerformReasonAnswer(CONSTANTS.labResultsDidNotPerformReasonQuestion));
       dispatch(constantsActions.fetchLabResultsTestLocationAnswer(CONSTANTS.labResultsTestLocationQuestion));
+      dispatch(patientAction.fetchPatientLabTestResults(state.patient.uuid));
```

Comment:
We've been doing `state.patient.uuid` in some places, we should agree on a standard place/way to get the patient data, most likely the store. passing in via the URL seems redundant.

Reply:
we're passing props from one page to route to the other page, so I think passing in the patientUUID in the route state ain't really much, the alternative would be fetching and setting a selectedPatient in redux, before actually routing, either is fine but I don't think it's redundant